parser/Builder.cc

```diff
@@ -470,8 +472,14 @@ class Builder::Impl {
             auto elseLoc = tokLoc(elseTok).join(maybe_loc(else_));
             else_stmts.emplace_back(std::move(else_));
             stmts.emplace_back(make_unique<Begin>(elseLoc, std::move(else_stmts)));
+            auto loc = collectionLoc(stmts);
+            if (!stmts.empty()) {
```

Comment:
Isn't this always true? `stmts` is extended on line 474, and `collecitonLoc` doesn't appear to clear out its argument.